framework/src/base/NonlinearSystem.C

```diff
@@ -680,16 +694,28 @@ NonlinearSystem::computeResidual(NumericVector<Number> & residual, Moose::Kernel
 
   try
   {
-    residual.zero();
```

Comment:
`_time_integrator->preStep()` cannot and should not be called on INITIAL because of `1/dt` operation with `dt==0`.